membership-attribute-service/app/controllers/HealthCheckController.scala

```diff
@@ -36,7 +36,7 @@ class HealthCheckController(touchPointBackends: TouchpointBackends, override val
       if (failures.isEmpty) {
         Ok(Json.obj("status" -> "ok", "gitCommitId" -> app.BuildInfo.gitCommitId))
       } else {
-        failures.flatMap(_.messages).foreach(msg => logger.warn(msg))
+        failures.flatMap(_.messages).foreach(msg => logger.warnNoPrefix(msg))
```

Comment:
we use `...NoPrefix` where there will never be an identity id available to log as it's not a user request.